skbio/util/_decorator.py

```diff
@@ -82,3 +85,207 @@ def __get__(self, cls, owner):
 
     def __set__(self, obj, value):
         raise AttributeError("can't set attribute")
+
+
+class _state_decorator(object):
+    """ Base class for decorators of all public functionality.
+    """
+
+    _line_prefix = '\n        '
```

Comment:
Can you add a comment describing the specific number of trailing spaces here? Or build `_line_prefix` in such a way that it's self-documenting.